File: SoloLearn/Paint Costs/PaintCosts.c
#include <stdio.h>
#include <math.h>

int main() {
    float colors, total, tax;
    scanf("%f", &colors);
    total = 40 + 5 * colors;
    tax = total / 10;
    printf("%d", (int) ceil(total + tax));
    return 0;
} 

File: SoloLearn/Code Coach/Paint Costs/PaintCosts.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            float colors = float.Parse(Console.ReadLine()), total = 40 + 5 * colors, tax = total / 10;
            Console.WriteLine(Convert.ToInt32(Math.Ceiling(total + tax)));
        }
    }
} 